fdbclient/MultiVersionTransaction.actor.cpp

```diff
@@ -1777,7 +1787,7 @@ void MultiVersionTenant::TenantState::updateTenant() {
 		tenant = Reference<ITenant>(nullptr);
 	}
 
-	tenantVar->set(tenant);
+	tenantVar->set(tenant, true);
```

Comment:
Are there cases where this would trigger unnecessarily (e.g. if everything is valid and the tenant is still the same)? I'm not sure if that can happen, but it would be nice to avoid failing in-flight work if it did.

Reply:
In needed specifically for the case when going from the initial state to an error state. So that abortable futures are triggered while still having an empty reference to the tenant. I can add a check to avoid triggering in other cases